src/Routes.js

```diff
@@ -90,82 +73,18 @@ const TemplateDetail = lazy(() =>
     )
 );
 
-export const Routes = () => {
-    const history = useHistory();
+const PatchRoutes = () => {
     const chrome = useChrome();
 
     const generalPermissions = ['patch:*:*', 'patch:*:read'];
     const [hasSystems, setHasSystems] = useState(true);
     const INVENTORY_TOTAL_FETCH_URL = '/api/inventory/v1/hosts';
 
-    const paths = [
-        {
-            path: '/advisories/:advisoryId/:inventoryId',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: InventoryDetail
-        },
-        {
-            path: '/advisories/:advisoryId',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: AdvisoryPage
-        },
-        {
-            path: '/advisories',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: Advisories
-        },
-        {
-            path: '/systems/:inventoryId',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: InventoryDetail
-        },
-        {
-            path: '/systems',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: Systems
-        },
-        {
-            path: '/packages/:packageName/:inventoryId',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: InventoryDetail
-        },
-        {
-            path: '/packages/:packageName',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: PackageDetail
-        },
-        {
-            path: '/packages',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: PackagesPage
-        },
-        {
-            path: '/templates/:templateName',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: TemplateDetail
-        },
-        {
-            path: '/templates',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: Templates
-        }
-    ];
-
     const listenNavigation = useCallback(() => {
         if (chrome) {
             return chrome.on('APP_NAVIGATION', event => {
                 if (event.domEvent) {
-                    history.push(`/${event.navId}`);
+                    chrome.appNavClick(`/${event.navId}`);
```

Comment:
According to the platform team, this whole part related to `APP_NAVIGATION` can and should be removed (see https://github.com/RedHatInsights/compliance-frontend/pull/1892/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67 for example)